### DISCUSSION TITLE
Implement Dark Mode feature.

### DISCUSSION THREAD

**Discussion by aritra-tech** [waseefakhtar/dose-android#55] [Ideas]
Right now it's showing the color of the device. So how about we implement the dark mode feature so that users can switch between the Dynamic Theme?

**Comment by waseefakhtar** [comment]
Hmm, doesn't it already switch to dark mode if you switch to dark mode on your device? 🤔

**Reply by aritra-tech** [reply]
Yeah it switched to dark mode

**Reply by waseefakhtar** [reply]
Perfect! :)
